src/main/webapp/map.html

```diff
@@ -24,41 +24,47 @@
       }
 
       //initializes the map with specified center and zoom
-      function createRestaurantsMap(){
-        fetch('/Restaurants').then(function(response) {
+      function createCountriesMap(){
+        fetch('/Countries').then(function(response) {
           return response.json();
-        }).then((FamousRestaurants) => {
+        }).then((CountryMaps) => {
+          
           var prev_infowindow = false;
 
             map = new google.maps.Map(document.getElementById('map'), {
-            center: {lat: 33.4774, lng: -81.97},
-            zoom: 7
+            center: {lat: 16.4774, lng: -24.97},
+            zoom: 3
           });
-          FamousRestaurants.forEach((FamousRestaurant) => {
+          CountryMaps.forEach((CountryMap) => {
+            var countryUrl = 'http://localhost:8080/country/' + CountryMap.code + '';
+            //console.log(CountryMap.lat);
              const marker = new google.maps.Marker({
-              position: {lat: FamousRestaurant.lat, lng: FamousRestaurant.lng},
+              position: {lat: CountryMap.lat, lng: CountryMap.lng},
               map: map
             });
-             var infoWindow = new google.maps.InfoWindow({
-              content: FamousRestaurant.state
-             });
              marker.addListener('click', function() {
+              window.location.href = countryUrl;
               //prevents multiple infowindows from being open at the same time
               if (prev_infowindow) {
                 prev_infowindow.close();
               }
               prev_infowindow = infoWindow;
               infoWindow.open(map, marker);
              });
+
+             var infoWindow = new google.maps.InfoWindow({
```

Comment:
How about moving this into the click handler since it doesn't look like it is really used outside of that function?